### ISSUE TITLE
Status ERROR: bndtools.core code=0 Cannot find ...

### ISSUE THREAD

**Issue by rotty3000** [bndtools/bnd#1509]
When attempting to start a bndrun config against a resolved project I get the following error:
```
Status ERROR: bndtools.core code=0 Cannot find /error/org.apache.xbean.asm-shaded;version=[3.8.0,3.8.1) Not found in [bnd-cache, MavenBndRepository [localRepo=/home/rotty/.m2/repository, storage=Central, inited=true], MavenBndRepository [localRepo=/home/rotty/.m2/repository, storage=Local, inited=true], Release, Distro] null
```
![run-osgi-error](https://cloud.githubusercontent.com/assets/146764/16208960/c376f81a-3702-11e6-9e11-4da8a6548dbb.png)

See https://github.com/rotty3000/com.rotty3000.ejb

**Comment by rotty3000** [comment]
The library does seem to be in my .m2 repo.

**Comment by rotty3000** [comment]
fixed by #1512
